x/did/tests/update_diddoc_test.go

```diff
@@ -613,4 +613,128 @@ var _ = Describe("DIDDoc update", func() {
 			})
 		})
 	})
+
+	Describe("Service: adding a new one", func() {
+		var alice CreatedDidDocInfo
+		var msg *types.MsgUpdateDidDocPayload
+
+		BeforeEach(func() {
+			alice = setup.CreateSimpleDid()
+
+			newAccepts := []string{"accept-1", "accept-2"}
+			newRoutingKeys := []string{"did:example:HPXoCUSjrSvWC53SLWQjsm#somekey", "did:example:HPXoCUSjrSvWC54SLWQjsm#somekey"}
```

Comment:
I'll double check the spec, but maybe we want to add a validation for keys uniqueness

Reply:
Example
```
validation.Field(&didDoc.Authentication,
				IsUniqueStrList(), validation.Each(IsSpecificDIDUrl(allowedNamespaces, Empty, Empty, Required), HasPrefix(didDoc.Id)),
		),
```

Reply:
Done! ✅